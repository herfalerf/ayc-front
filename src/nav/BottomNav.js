import React from "react";
import { NavLink } from "react-router-dom";
import { Link as RouterLink } from "react-router-dom";
import { makeStyles } from "@material-ui/core/styles";
import {
  Typography,
  Grid,
  List,
  ListItem,
  ListItemText,
  Icon,
  Link,
} from "@material-ui/core";
import LinkedInIcon from "@material-ui/icons/LinkedIn";
import FacebookIcon from "@material-ui/icons/Facebook";
import PhoneIcon from "@material-ui/icons/Phone";
import MailOutlineIcon from "@material-ui/icons/MailOutline";
import CopyrightIcon from "@material-ui/icons/Copyright";

// Bottom sitemap navigation.  Shows up on every page, contains links to social media as well as site links and link for admin access
//
// Rendered by App

const useStyles = makeStyles((theme) => ({
  botNav: {
    marginTop: 0,
    backgroundImage: `url('${process.env.PUBLIC_URL}/images/nav/nav.png')`,
    backgroundSize: "cover",
    backgroundPosition: "center center",
    // height: "120vh",
    minHeight: "25vh",
  },
  h1: {
    marginTop: 0,
  },
  botLogoItem: {
    padding: theme.spacing(2),
  },
  botItem: {
    padding: theme.spacing(2),
  },

  botLogo: {
    maxHeight: 40,
  },
  textWhite: {
    color: "#FFFFFF",
  },
  textPrimary: {
    color: "#00c7d7",
  },
}));

const BottomNav = () => {
  const classes = useStyles();
  return (
    <div className={classes.botNav}>
      <div>
        <Grid container>
          <Grid
            container
            alignItems="center"
            justifyContent="flex-start"
            item
            xs={6}
            className={classes.botItem}
          >
            <Grid item xs={8} classname={classes.botItem}>
              {" "}
              <img
                src={process.env.PUBLIC_URL + "/images/logos/Logo03b.png"}
                className={classes.botLogo}
              ></img>
              <Typography className={classes.textWhite}>
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Lorem
                ipsum dolor sit amet, consectetur adipisicing elit.
              </Typography>
            </Grid>
            {/* <Grid item xs={6}></Grid> */}
          </Grid>
          <Grid item xs={2} className={classes.botItem}>
            <List>
              <ListItem>
                <ListItemText
                  className={classes.textPrimary}
                  primary="QUICK LINKS"
                ></ListItemText>
              </ListItem>
              <ListItem>
                <Link component={RouterLink} to="/">
                  <ListItemText className={classes.textWhite} primary="Home" />
                </Link>
              </ListItem>
              <ListItem>
                <Link component={RouterLink} to="/methods">
                  <ListItemText
                    className={classes.textWhite}
                    primary="Methodology"
                  />
                </Link>
              </ListItem>
              <ListItem>
                <Link component={RouterLink} to="/about">
                  {" "}
                  <ListItemText
                    className={classes.textWhite}
                    primary="About Us"
                  />
                </Link>
              </ListItem>
              <ListItem>
                <Link component={RouterLink} to="/contact">
                  {" "}
                  <ListItemText
                    className={classes.textWhite}
                    primary="Contact"
                  />
                </Link>
              </ListItem>
            </List>
          </Grid>
          <Grid item xs={2} className={classes.botItem}>
            <List>
              <ListItem className={classes.textPrimary}>
                <ListItemText primary="OUR SERVICES" />
              </ListItem>
              <ListItem>
                <Link component={RouterLink}>
                  {" "}
                  <ListItemText
                    className={classes.textWhite}
                    primary="Workshops"
                  />
                </Link>
              </ListItem>
              <ListItem>
                <Link>
                  {" "}
                  <ListItemText
                    className={classes.textWhite}
                    primary="Culture Assessments"
                  />
                </Link>
              </ListItem>
              <ListItem>
                <Link>
                  <ListItemText
                    className={classes.textWhite}
                    primary="Consulting"
                  />
                </Link>
              </ListItem>
            </List>
          </Grid>
          <Grid item xs={2} className={classes.botItem}>
            <List>
              <ListItem className={classes.textPrimary}>
                <ListItemText primary="RESOURCES" />
              </ListItem>
              <ListItem>
                <Link>
                  {" "}
                  <ListItemText
                    className={classes.textWhite}
                    primary="Video Library"
                  />
                </Link>
              </ListItem>
            </List>
          </Grid>

          <Grid container>
            <Grid item xs={3} className={classes.botItem}>
              <Typography className={classes.textWhite}>
                <PhoneIcon />
                1-800-123-1234
              </Typography>
            </Grid>

            <Grid item xs={3} className={classes.botItem}>
              <Link component="a" href="mailto:test@email.com">
                <Typography className={classes.textWhite}>
                  <MailOutlineIcon />
                  test@email.com
                </Typography>
              </Link>
            </Grid>
            <Grid item xs={1} className={classes.botItem}>
              <Link component="a" href="https://www.facebook.com">
                <FacebookIcon className={classes.textWhite} />
              </Link>
            </Grid>
            <Grid item xs={1} className={classes.botItem}>
              <Link componet="a" href="https://www.linkedin.com">
                <LinkedInIcon className={classes.textWhite} />
              </Link>
            </Grid>
            <Grid item xs={3} className={classes.botItem}>
              <Typography className={classes.textWhite}>
                <CopyrightIcon />
                2021 AYC. All Rights Reserved
              </Typography>
            </Grid>
          </Grid>
        </Grid>
      </div>

      {/* <NavLink to="/admin">Admin</NavLink> */}
    </div>
  );
};

export default BottomNav;
