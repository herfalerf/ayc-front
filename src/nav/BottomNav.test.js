import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import BottomNav from "./BottomNav";

it("renders without crashing", function () {
  render(
    <MemoryRouter>
      <BottomNav />
    </MemoryRouter>
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <BottomNav />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
