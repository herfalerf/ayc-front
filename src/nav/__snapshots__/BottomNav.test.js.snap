// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="makeStyles-botNav-9"
    >
      <div>
        <div
          class="MuiGrid-root MuiGrid-container"
        >
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-center MuiGrid-grid-xs-6"
          >
            <div
              class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-8"
            >
               
              <img
                alt="AYC Logo"
                class="makeStyles-botLogo-14"
                src="/images/logos/Logo03b.png"
              />
              <p
                class="MuiTypography-root makeStyles-textWhite-15 MuiTypography-body1"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-2"
          >
            <ul
              class="MuiList-root MuiList-padding"
            >
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <div
                  class="MuiListItemText-root makeStyles-textPrimary-16"
                >
                  <span
                    class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                  >
                    QUICK LINKS
                  </span>
                </div>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="/"
                >
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Home
                    </span>
                  </div>
                </a>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="/methods"
                >
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Methodology
                    </span>
                  </div>
                </a>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="/about"
                >
                   
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      About Us
                    </span>
                  </div>
                </a>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                  href="/contact"
                >
                   
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Contact
                    </span>
                  </div>
                </a>
              </li>
            </ul>
          </div>
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-2"
          >
            <ul
              class="MuiList-root MuiList-padding"
            >
              <li
                class="MuiListItem-root makeStyles-textPrimary-16 MuiListItem-gutters"
              >
                <div
                  class="MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                  >
                    OUR SERVICES
                  </span>
                </div>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                >
                   
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Workshops
                    </span>
                  </div>
                </a>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                >
                   
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Culture Assessments
                    </span>
                  </div>
                </a>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                >
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Consulting
                    </span>
                  </div>
                </a>
              </li>
            </ul>
          </div>
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-2"
          >
            <ul
              class="MuiList-root MuiList-padding"
            >
              <li
                class="MuiListItem-root makeStyles-textPrimary-16 MuiListItem-gutters"
              >
                <div
                  class="MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                  >
                    RESOURCES
                  </span>
                </div>
              </li>
              <li
                class="MuiListItem-root MuiListItem-gutters"
              >
                <a
                  class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                >
                   
                  <div
                    class="MuiListItemText-root makeStyles-textWhite-15"
                  >
                    <span
                      class="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
                    >
                      Video Library
                    </span>
                  </div>
                </a>
              </li>
            </ul>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container"
          >
            <div
              class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-3"
            >
              <p
                class="MuiTypography-root makeStyles-textWhite-15 MuiTypography-body1"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
                  />
                </svg>
                1-800-123-1234
              </p>
            </div>
            <div
              class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-3"
            >
              <a
                class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                href="mailto:test@email.com"
              >
                <p
                  class="MuiTypography-root makeStyles-textWhite-15 MuiTypography-body1"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"
                    />
                  </svg>
                  test@email.com
                </p>
              </a>
            </div>
            <div
              class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-1"
            >
              <a
                class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                href="https://www.facebook.com"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-textWhite-15"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                  />
                </svg>
              </a>
            </div>
            <div
              class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-1"
            >
              <a
                class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
                componet="a"
                href="https://www.linkedin.com"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root makeStyles-textWhite-15"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                  />
                </svg>
              </a>
            </div>
            <div
              class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-3"
            >
              <p
                class="MuiTypography-root makeStyles-textWhite-15 MuiTypography-body1"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M11.88 9.14c1.28.06 1.61 1.15 1.63 1.66h1.79c-.08-1.98-1.49-3.19-3.45-3.19C9.64 7.61 8 9 8 12.14c0 1.94.93 4.24 3.84 4.24 2.22 0 3.41-1.65 3.44-2.95h-1.79c-.03.59-.45 1.38-1.63 1.44-1.31-.04-1.86-1.06-1.86-2.73 0-2.89 1.28-2.98 1.88-3zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
                  />
                </svg>
                2021 AYC. All Rights Reserved
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="makeStyles-botNavMobile-10"
    >
      <div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column"
        >
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-6"
          >
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="tel:1231231234"
            >
              <p
                class="MuiTypography-root makeStyles-textWhite-15 MuiTypography-body1"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
                  />
                </svg>
                123-123-1234
              </p>
            </a>
          </div>
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-6"
          >
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="mailto:test@email.com"
            >
              <p
                class="MuiTypography-root makeStyles-textWhite-15 MuiTypography-body1"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8l8 5 8-5v10zm-8-7L4 6h16l-8 5z"
                  />
                </svg>
                test@email.com
              </p>
            </a>
          </div>
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-6"
          >
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="https://www.facebook.com"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-textWhite-15"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M5 3h14a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2m13 2h-2.5A3.5 3.5 0 0 0 12 8.5V11h-2v3h2v7h3v-7h3v-3h-3V9a1 1 0 0 1 1-1h2V5z"
                />
              </svg>
            </a>
          </div>
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-6"
          >
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              componet="a"
              href="https://www.linkedin.com"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root makeStyles-textWhite-15"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                />
              </svg>
            </a>
          </div>
          <div
            class="MuiGrid-root makeStyles-botItem-13 MuiGrid-item MuiGrid-grid-xs-12"
          >
            <p
              class="MuiTypography-root makeStyles-textWhite-15 MuiTypography-body1"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.88 9.14c1.28.06 1.61 1.15 1.63 1.66h1.79c-.08-1.98-1.49-3.19-3.45-3.19C9.64 7.61 8 9 8 12.14c0 1.94.93 4.24 3.84 4.24 2.22 0 3.41-1.65 3.44-2.95h-1.79c-.03.59-.45 1.38-1.63 1.44-1.31-.04-1.86-1.06-1.86-2.73 0-2.89 1.28-2.98 1.88-3zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
                />
              </svg>
              2021 AYC. All Rights Reserved
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
